training/06-inner-blocks.md

```diff
@@ -124,29 +124,10 @@ First, look at the [`package.json`](https://github.com/10up/gutenberg-lessons/tr
     }
 ```
 
-Second, we need to register the block in [`blocks.php`](https://github.com/10up/gutenberg-lessons/tree/trunk/themes/tenup-theme/includes/blocks.php). There are two steps to this. Step one: find the comment `// Require custom blocks` and the line `require_once TENUP_THEME_BLOCK_DIR . '/inner-blocks-one-complete/register.php';`. We want to duplicate this line and reference the starter register file:
+Second, we need to set the `editorScript` property in [`block.json`](https://github.com/10up/gutenberg-lessons/blob/trunk/themes/tenup-theme/includes/blocks/inner-blocks-one-starter/block.json)
```

Comment:
Not sure if I'm missing something here...maybe this `editorScript` property is added by default when running a command that I missed (??), but I had to add it manually in order to make it work. Any thoughts?